### PR TITLE
main <- Develop miv soft cpu

### PR DESCRIPTION
**Pull request by CLappin**
Creating pull request to merge Develop_miv_soft_cpu to main.
Github.io page will be updated after pull request #18 has been merged.

### CODE REVIEW

File: _docs/3-mi-v-soft-cpu.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+permalink: /docs/mi-v-soft-cpu/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Mi-V Soft CPU Page."
+last_modified_at: 2018-01-10T11:22:01-05:00
+---
+### Mi-V Soft CPUs
+Mi-V Ecosystem supports following RV32I base instruction set processors with optional extensions.
```
Comment (kevinscully100) [inline _docs/3-mi-v-soft-cpu.md]: The Mi-V Ecosystem supports the following RV32I base instruction set processors with optional extensions.

File: _docs/3-mi-v-soft-cpu.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* MIV_RV32
+* MIV_RV32IMA_L1_AHB * 
+* MIV_RV32IMA_L1_AXI * 
+* MIV_RV32IMAF_L1_AHB *
+
+\*Legacy cores. MIV_RV32 recommended for new designs
```
Comment (kevinscully100) [inline _docs/3-mi-v-soft-cpu.md]: Can this be aligned with the list items? i.e. align * with M in MIV_

Add a full stop.

File: _docs/3-mi-v-soft-cpu.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  
+### Hardware platforms and boards
+Mi-V Ecosystem supports following hardware platforms.
```
Comment (kevinscully100) [inline _docs/3-mi-v-soft-cpu.md]: The Mi-V Ecosystem supports the following hardware platforms.

File: _docs/3-mi-v-soft-cpu.md (hunk 1 above)
Comment (CLappin) [reply _docs/3-mi-v-soft-cpu.md]: Agreed, will make this change now.

File: _docs/3-mi-v-soft-cpu.md (hunk 2 above)
Comment (CLappin) [reply _docs/3-mi-v-soft-cpu.md]: I can try this now. And will add the full stop.

File: _docs/3-mi-v-soft-cpu.md (hunk 3 above)
Comment (CLappin) [reply _docs/3-mi-v-soft-cpu.md]: Agreed, will make this change now.

File: _docs/3-mi-v-soft-cpu.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+last_modified_at: 2018-01-10T11:22:01-05:00
+---
+### Mi-V Soft CPUs
+The Mi-V Ecosystem supports the following RV32I base instruction set processors with optional extensions.
+* MIV_RV32
+* MIV_RV32IMA_L1_AHB * 
+* MIV_RV32IMA_L1_AXI * 
+* MIV_RV32IMAF_L1_AHB *
+
+  \*Legacy cores. MIV_RV32 recommended for new designs
```
Comment (nitindeshpande) [inline _docs/3-mi-v-soft-cpu.md]: Not sure how this line is going to appear on rendered page.

Comment (kevinscully100) [reply _docs/3-mi-v-soft-cpu.md]: looks ok to me on the rendered view

**Pull request merged**